update_api_files.sh

```diff
@@ -74,6 +75,8 @@ echo "]===]))" >> "$FILENAME"
 # Copy the individual API files
 for f in "${file_list[@]}"; do
   source_file=$SOURCE/apis/$f.normal.json
+  jq 'walk( if (type == "object") and has("documentation") and (.documentation|contains("wJalrXUtnFEMI")) then del(.documentation) else . end )' "$source_file" >| "$TFILE"
```

Comment:
```suggestion
  # remove example keys from documentation to prevent security reports from being triggered
  jq 'walk( if (type == "object") and has("documentation") and (.documentation|contains("wJalrXUtnFEMI")) then del(.documentation) else . end )' "$source_file" >| "$TFILE"
```